src/net.cpp

```diff
@@ -606,34 +606,21 @@ void CNode::copyStats(CNodeStats &stats, const std::vector<bool> &m_asmap)
 }
 #undef X
 
-/**
```

Comment:
+1, this comment for a method  belongs in the header not in the implementation file.